lib/nerves_motd.ex

```diff
@@ -84,6 +84,7 @@ defmodule NervesMOTD do
     [
       [{"Uptime", uptime()}],
       [{"Clock", Utils.formatted_local_time()}],
+      [{"Temperature", runtime_mod().cpu_temperature()}],
```

Comment:
I don't think this will work on most Nerves devices. Which ones have you tested it on? Perhaps it should be optional and only show up if available.

Reply:
I tested on `grisp2` and `rpi0`. My understanding is the CPU temperature file is a Linux feature. But I am not sure about other boards. We could make it optional to be safe.